library/agent/context/user.ts

```diff
@@ -85,3 +80,18 @@ function logWarningSetUserCalledAfterMiddleware() {
 
   loggedWarningSetUserCalledAfterMiddleware = true;
 }
+
+let loggedWarningSetUserCalledWithoutContext = false;
+
+function logWarningSetUserCalledWithoutContext() {
+  if (loggedWarningSetUserCalledWithoutContext) {
+    return;
+  }
+
+  // eslint-disable-next-line no-console
+  console.warn(
+    `setUser(...) did not find a context. Make sure to call setUser(...) within a handler or middleware of a supported framework.`
```

Comment:
Let's align message with this one: https://github.com/AikidoSec/firewall-node/pull/463/files#diff-c27fd05be9ba283c1c1a6eecbcb5398aa4617a8fc23e248b0cb1a839ee242c5dR75? 💭

Reply:
Changed.